dll/win32/browseui/utility.cpp

```diff
@@ -1,4 +1,7 @@
 #include "precomp.h"
+#ifndef SHCIDS_CANONICALONLY
+#define SHCIDS_CANONICALONLY 0x10000000L
```

Comment:
It looks like this (and the two similar defines in ntobjshex header) could be moved to either a public header (if this is public) or one of our undoc_XXXX.h headers if not.

Reply:
This is a Microsoft documented define.

Reply:
The question was about whether there should be a better centralized place (header file) for such definition. Our situation with `SHCIDS_` consts is not very organized: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=SHCIDS_ (meanwhile MSDN mentions `shobjidl.h` and `shobjidl_core.h`)

Reply:
> should be a better centralized place

There is, where IShellFolder2 is defined. But I was reluctant to change those other unrelated shell extensions in this PR.

Reply:
No problem. If you feel like it you can make the changes in a separate PR.